src/test/java/com/bytezone/dm3270/TerminalClientTest.java

```diff
@@ -101,35 +101,6 @@ private void setServiceFlowFromFile(String s) throws FileNotFoundException {
     service.setFlow(Flow.fromYml(new File(getResourceFilePath(s))));
   }
 
-  private static class ExceptionWaiter implements ConnectionListener {
```

Comment:
any particular reason to move this? was here following the convention of following order of usage.